tests/Composer/Test/Satis/PackagesBuilderDumpTest.php

```diff
@@ -23,6 +23,26 @@
  */
 class PackagesBuilderDumpTest extends \PHPUnit_Framework_TestCase
 {
+    protected $package;
+
+    protected $root;
+
+    protected function setUp()
+    {
+        $this->package = new Package('vendor/name', '1.0.0.0', '1.0');
+
+        $this->root = $this->setFileSystem();
+    }
+
+    protected function setFileSystem()
```

Comment:
Well, ok about side-effect/change state, but I didn't find a way to change the code that way. Then, i simplified removing the method...